frontend/packages/dev-console/src/components/__tests__/ProjectsExistWrapper.spec.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { FirehoseResult, LoadingBox } from '@console/internal/components/utils';
+import ProjectsExistWrapper from '../ProjectsExistWrapper';
+
+let projects: FirehoseResult;
+
+describe('ProjectsExistWrapper', () => {
+  beforeEach(() => {
+    projects = {
+      data: [],
+      loaded: true,
+      loadError: '',
+    };
+  });
+
+  it('should show loading box', () => {
+    projects.loaded = false;
+    const wrapper = shallow(
+      <ProjectsExistWrapper title="Topology" projects={projects}>
+        {}
```

Comment:
Why are you rendering an empty block of children instead of using a self closing tag? What's the point of `{}`

Reply:
I fixed it. I was thinking of something else.